test/testsuites/examples/create-eks.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-eks-task
+  namespace: tekton
```

Comment:
Gotcha -- choose a different name than `tekton` then, since it reads like a system namespace. Perhaps just use the default namespace until you have isolation requirements.